src/main/fc/fc_init.c

```diff
@@ -520,6 +505,21 @@ void init(void)
     motorDevInit(&motorConfig()->dev, idlePulse, getMotorCount());
     systemState |= SYSTEM_STATE_MOTORS_READY;
 
+    LED1_ON;
+    LED0_OFF;
+    LED2_OFF;
+
+    for (int i = 0; i < 10; i++) {
+        LED1_TOGGLE;
+        LED0_TOGGLE;
+        delay(25);
+        if (!(getBeeperOffMask() & (1 << (BEEPER_SYSTEM_INIT - 1)))) BEEP_ON;
+        delay(25);
+        BEEP_OFF;
+    }
+    LED0_OFF;
+    LED1_OFF;
+
 #ifdef USE_SERVOS
```

Comment:
Looking at the changes done in #4218, I'd move the LED blinking to after the servo init, just to revert the timing back to how it was before #4218 (or as close as possible).

Reply:
@mikeller Yes,this spurious pulse will affect servos too, i moved it after ..